docs/SUMMARY.md

```diff
@@ -35,17 +35,17 @@
 ## Contribute
 
 * [Code of Conduct](code-of-conduct.md)
-* [How to contribute](./contributor-guide.md)
-    * [Example pipelines](./contribute-example.md)
-* [Monorepo](coming-soon.md)
-* [Code style](coming-soon.md)
+* [How to contribute](contributor-guide.md)
+    * [Example pipelines](contribute-example.md)
+* [Monorepo](monorepo.md)
+* [Code style](code-style.md)
```

Comment:
The `coming-soon.md` page, once included and associated with the `Monorepo` section, would not render anymore for any of the subsequent inclusions on this page, resulting in non-links:
![image](https://user-images.githubusercontent.com/1894533/193695811-64a4c00e-9e36-45e1-8502-8b5aebae8a9b.png)

This is actually [documented by GitBook](https://docs.gitbook.com/getting-started/git-sync/content-configuration):
<img width="785" alt="image" src="https://user-images.githubusercontent.com/1894533/193695883-78f8d107-a9b5-4f24-86ab-60eb073a88c8.png">

The solution is to create individual coming-soon pages for all the appropriate entries, that will be filled in in the future.